### DISCUSSION TITLE
Zoraxy API documentation

### DISCUSSION THREAD

**Discussion by a0ndr** [tobychui/zoraxy#191] [Ideas]
I'd really like to use an API to create and manage routes. The Control Panel already uses an API but I couldn't find any documentation.

**Comment by tobychui** [comment]
I didn't got the time and reasons to write an API doc because
1. I thought most beginner user will just use the Web UI for management tasks
2. If you need API control for creating / managing route, there should be better options out there
 
Besides, current API authentications are based on browser cookies. Even if I write a doc for it, you still need to emulate a browser to access them, which doesn't make sense from a developer point of view. 

Maybe if someone contributed a token management system into Zoraxy as an alternative authentication method, then I can integrate the API function handler with it. Then after all these are done and pushed to release, that is when we need a proper API doc.
